library/filesys.po

```diff
@@ -1,60 +1,65 @@
-#: ../../library/filesys.rst:7
-msgid ""
-"The modules described in this chapter deal with disk files and directories.  "
-"For example, there are modules for reading the properties of files, "
-"manipulating paths in a portable way, and creating temporary files.  The "
-"full list of modules in this chapter is:"
-msgstr ""
-
-#: ../../library/filesys.rst:31
-msgid "Module :mod:`os`"
-msgstr ""
-
-#: ../../library/filesys.rst:30
-msgid ""
-"Operating system interfaces, including functions to work with files at a "
-"lower level than Python :term:`file objects <file object>`."
-msgstr ""
-
-#: ../../library/filesys.rst:35
-msgid "Module :mod:`io`"
-msgstr ""
-
-#: ../../library/filesys.rst:34
-msgid ""
-"Python's built-in I/O library, including both abstract classes and some "
-"concrete classes such as file I/O."
-msgstr ""
-
-#: ../../library/filesys.rst:37
-msgid "Built-in function :func:`open`"
-msgstr ""
-
-#: ../../library/filesys.rst:38
-msgid "The standard way to open files for reading and writing with Python."
-msgstr ""
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2001-2021, Python Software Foundation
+# This file is distributed under the same license as the Python package.
+#
+# Translators:
+# Leon H., 2017
+# Allen Wu <allen91.wu@gmail.com>, 2021
+msgid ""
+msgstr ""
+"Project-Id-Version: Python 3.10\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-06-20 18:08+0800\n"
+"PO-Revision-Date: 2021-11-22 20:13+0800\n"
+"Last-Translator: Allen Wu <allen91.wu@gmail.com>\n"
+"Language-Team: Chinese - TAIWAN (https://github.com/python/python-docs-zh-"
+"tw)\n"
+"Language: zh_TW\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+"X-Generator: Poedit 3.0\n"
+
+#: ../../library/filesys.rst:5
+msgid "File and Directory Access"
+msgstr "檔案與目錄存取"
+
+#: ../../library/filesys.rst:7
+msgid ""
+"The modules described in this chapter deal with disk files and directories.  "
+"For example, there are modules for reading the properties of files, "
+"manipulating paths in a portable way, and creating temporary files.  The "
+"full list of modules in this chapter is:"
+msgstr ""
+"本章中描述的 module（模組）用於處理硬碟檔案和目錄。 例如，有一些 module（模組）用於讀取檔案的屬性、以可攜"
+"（portable）方式操作路徑以及建立暫存檔。 本章中的完整 module（模組）清單是："
+
+#: ../../library/filesys.rst:31
+msgid "Module :mod:`os`"
+msgstr "Module（模組）\\ :mod:`os`"
+
+#: ../../library/filesys.rst:30
+msgid ""
+"Operating system interfaces, including functions to work with files at a "
+"lower level than Python :term:`file objects <file object>`."
+msgstr ""
+"作業系統介面中，包括比 Python 更低階的檔案處理函式：\\ :term:`檔案物件 <file object>` 。"
+
+#: ../../library/filesys.rst:35
+msgid "Module :mod:`io`"
+msgstr "Module（模組）\\ :mod:`io`"
+
+#: ../../library/filesys.rst:34
+msgid ""
+"Python's built-in I/O library, including both abstract classes and some "
+"concrete classes such as file I/O."
+msgstr "Python 的內建 I/O 函式庫，包含抽象類別和一些具體類別（concrete class），如檔案 I/O。"

```

Comment:
Use English punctuation here since the content inside `()` is in pure English.

```suggestion
msgstr "Python 的內建 I/O 函式庫，包含抽象類別和一些具體類別 (concrete class)，如檔案 I/O。"
```